tests/integration/convert-offline-systems/prepare_system.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+from envparse import env
+
+
+def test_prepare_system(shell):
+    assert shell("yum install dnsmasq -y").returncode == 0
+
+    # Install katello package
+    pkg_url = "https://dogfood.sat.engineering.redhat.com/pub/katello-ca-consumer-latest.noarch.rpm"
+    pkg_dst = "/usr/share/convert2rhel/subscription-manager/katello-ca-consumer-latest.noarch.rpm"
+    assert shell("wget --no-check-certificate --output-document {} {}".format(pkg_dst, pkg_url)).returncode == 0
+    assert shell("rpm -i {}".format(pkg_dst)).returncode == 0
+
+    satellite_ip = shell("dig +short dogfood.sat.engineering.redhat.com | tail -1").output
+    sbrmng_ip = shell("dig +short dhcp-8-29-228.lab.eng.rdu2.redhat.com | tail -1").output
+
+    with open("/etc/dnsmasq.conf", "a") as f:
+        # Satellite url
+        f.write("address=/dogfood.sat.engineering.redhat.com/{}".format(satellite_ip))
+
+        # Url for sub manager
+        f.write("address=/dhcp-8-29-228.lab.eng.rdu2.redhat.com/{}".format(sbrmng_ip))
```

Comment:
I found this URL when looking into `/var/log/rhsm/rhsm.log`. Sub manager tries to register to this url when katello package is installed. 

However as you say it doesn't look good. I only found this url here https://dogfood.sat.engineering.redhat.com/pub/ (package names). But it still doesn't mean it will not change..

Sadly I have no idea what other way to do it right now.

Reply:
I don't even remember why do we need to allow access to some subscription-manager-related hostname. Allowing access to the Satellite server only should be sufficient. The reason is that after installing the katello package, the pre-installed subscription-manager should be communicating only with the Satellite server.

Try removing this line and let's see what happens :)

Reply:
I created a jira ticket for this integration test: https://issues.redhat.com/browse/OAMG-6426
I posted the findings there.

Reply:
Result: we decided to replace all "dhcp" URLs to "dogfood" inside the `rhsm.conf` file.